deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -574,6 +593,48 @@ def detect_windows_whodata_mode_change(file_monitor, file='.*'):
                        error_message=ERR_MSG_WHODATA_REALTIME_MODE_CHANGE_EVENT)
 
 
+def detect_audit_queue_full(file_monitor, update_position=True):
+    """Detects the configured value for the whodata queue
+    Args:
+        file_monitor (FileMonitor): file log monitor to detect events
+        update_position (bool, optional): True if we pop items from the queue once they are read. False otherwise.
+                                          Default `True`
+    """
+
+    return file_monitor.start(timeout=T_10, callback=generate_monitoring_callback(CB_WHODATA_QUEUE_FULL),
+                              error_message=create_error_message(CB_WHODATA_QUEUE_FULL),
+                              update_position=update_position).result()
+
+
+def detect_invalid_conf_value(file_monitor, element):
+    """Detects the error generated when a configuration element has an invalid value.
+    Args:
+        file_monitor (FileMonitor): file log monitor to detect events
+        element (str): Element name that is being detected
+    """
+    pattern = fr".*Invalid value for element (\'{element}\': .*)"
+    return file_monitor.start(timeout=T_10, callback=generate_monitoring_callback(pattern),
+                              error_message=create_error_message(pattern)).result()
+
+
+def detect_audit_healthcheck_failed(file_monitor):
+    """Detects if the initial audit healtcheck has failed
+    Args:
+        file_monitor (FileMonitor): file log monitor to detect events
+    """
+    return file_monitor.start(timeout=T_10, callback=generate_monitoring_callback(CB_AUDIT_HEALTHCHECK_FAILED),
+                              error_message=create_error_message(CB_AUDIT_HEALTHCHECK_FAILED)).result()
+
+
+def get_configured_whodata_queue_size(file_monitor):
+    """Detects the configured value for the whodata queue

```

Comment:
Line 597 and 630 have the same description.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4399/commits/98eac147dbd7032162e0d59b897326aeace08baa